pwnlib/elf/corefile.py

```diff
@@ -1294,12 +1295,42 @@ def apport_read_crash_data(self):
 
         return data
 
+    def systemd_coredump_corefile(self):
+        """Find the systemd-coredump crash for the process and dump it to a file.
+
+        Arguments:
+            process(process): Process object we're looking for.
+
+        Returns:
+            `str`: Filename of core file, if coredump was found.
+        """
+        filename = "core.%s.%i.coredumpctl" % (self.basename, self.pid)
+        try:
+            subprocess.check_call(
+                [
+                    "coredumpctl",
+                    "dump",
+                    "--output=%s" % filename,
+                    # Filter coredump by pid and filename
+                    str(self.pid),
+                    self.basename,
+                ],
+                stdout=open(os.devnull, 'w'),
+                stderr=open(os.devnull, 'w'),
```

Comment:
I would suggest replacing the doubled `open()` call with stderr set to `STDOUT`
```suggestion
                stderr=subprocess.STDOUT,
```